message/tests.py

```diff
@@ -0,0 +1,55 @@
+from django.utils import timezone
+from .models import Message
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+from .models import AuthorOptions
+import pytest
+
+AUTHOR = 'testuser@gmail.com'
+DOG_OWNER_ID = '123456789'
+DAYCARE_ID = '987654321'
+TEXT = 'Hello this is the test message from owner to day care'
+DATE = timezone.now()
+
+
+@pytest.fixture
+def create_message():
+    return Message.create(author=AuthorOptions.DogOwner,
+                          dogowner_id=DogOwner.objects.get(pk=1),
+                          daycare_id=DayCare.objects.get(pk=1),
+                          text=TEXT)
+
+
+@pytest.fixture
+def create_message2():
+    return Message.create(author=AuthorOptions.DayCare,
+                          dogowner_id=DogOwner.objects.get(pk=1),
+                          daycare_id=DayCare.objects.get(pk=1),
+                          text='This is reply to first message from daycare to owner')
```

Comment:
The names of are confusing, it is not clear who is the sender, who is the recipient and if its a replay or not
Please add more fixtures instead of piggy backing this one, for example:
dogowner_message1()
daycare_replay_to_dogonwer_message1()
daycare_message1()
dogowner_replay_to_daycare_message1()

Reply:
fixed names and added more fixtures